src/emotes.ts

```diff
@@ -0,0 +1,107 @@
+import { emotes } from './emote_list';
+
+/* -------------------- HELPER FUNCTIONS -------------------- */
+
```

Comment:
I'd put the helper functions in a separate file called `utils.ts`, it's typically good practice to separate contexts into different files